sdk/dotnet/AppAutoScaling/Inputs/ScheduledActionScalableTargetActionArgs.cs

```diff
@@ -13,16 +13,16 @@ namespace Pulumi.Aws.AppAutoScaling.Inputs
     public sealed class ScheduledActionScalableTargetActionArgs : Pulumi.ResourceArgs
     {
         /// <summary>
-        /// The maximum capacity.
+        /// The maximum capacity. At least one of `max_capacity` or `min_capacity` must be set.
         /// </summary>
         [Input("maxCapacity")]
-        public Input<int>? MaxCapacity { get; set; }
+        public Input<string>? MaxCapacity { get; set; }
 
         /// <summary>
-        /// The minimum capacity.
+        /// The minimum capacity. At least one of `min_capacity` or `max_capacity` must be set.
         /// </summary>
         [Input("minCapacity")]
-        public Input<int>? MinCapacity { get; set; }
+        public Input<string>? MinCapacity { get; set; }
```

Comment:
This is breaking, let's triage

Reply:
So they changed this to allow the equivalent of nullable :/ 

Do we want to accept that or do we want to try and back that out?

Reply:
@mikhailshilkov I just updated this as per the old workings

Reply:
But... now it is a `bool` somehow?

Reply:
*NOW* it's fixed :) That was my derp!